README.md

```diff
@@ -46,13 +46,12 @@ tox -e dev
 ### How to use with PyCharm
 
 1. Create a new project using existing sources with your local working copy of this repository as root directory. Choose
```

Comment:
Ich verstehe ehrlich gesagt nur Bahnhof, weiß aber nicht, ob es anderen auch so gehen würde. Ich hätte Folgendes in ganz einfacher Sprache verstanden: 
You have cloned the repository, you want to work in, and have created the virtual environment, in which the repository should be executed. Now you have to make the repository _actually_ work inside this virtual environment. This means you need to connect your repository with the virtual environment - to be more precise: with the interpreter of this environment. How to do this:  
a) navigate to: File -> Settings (Strg + Alt + S) -> Project: your_project -> Python Interpreter -> Add interpreter -> Existing
b) Choose as interpreter: your_repo\ tox\dev\Scripts\python.exe